core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -165,14 +174,30 @@ class AdminClient(val time: Time,
                                   consumers: Option[List[ConsumerSummary]],
                                   coordinator: Node)
 
-  def describeConsumerGroup(groupId: String): ConsumerGroupSummary = {
-    val coordinator = findCoordinator(groupId)
+  def describeConsumerGroupHandler(coordinator: Node, groupId: String): GroupMetadata = {
     val responseBody = send(coordinator, ApiKeys.DESCRIBE_GROUPS,
         new DescribeGroupsRequest.Builder(Collections.singletonList(groupId)))
     val response = responseBody.asInstanceOf[DescribeGroupsResponse]
     val metadata = response.groups.get(groupId)
     if (metadata == null)
       throw new KafkaException(s"Response from broker contained no metadata for group $groupId")
+    metadata
+  }
+
+  def describeConsumerGroup(groupId: String): ConsumerGroupSummary = describeConsumerGroup(groupId, 0)
```

Comment:
I slightly prefer to use a default argument, but I don't have a strong feeling. Either way, usages of this will go away anyway when we have the real AdminClient (assuming that will actually happen).